src/core/initializeComponentsFactory.js

```diff
@@ -50,8 +50,7 @@ export default (
     componentRegistry.register(namespace, component);
   });
 
-  // Output the finalized configuration
-  logger.info("Runtime configuration:\n", JSON.stringify(config, null, 2));
+  logger.log("Computed configuration:", config.toJSON());
```

Comment:
Will this break the format of the JSON in the console?

Reply:
What do you mean by "break the format"? `toJSON` returns an object so the browser will just log it as an object rather than a string.

Reply:
I guess logging an object is good as well; they can expand it.

![Screen Shot 2019-06-13 at 5 34 08 PM](https://user-images.githubusercontent.com/1738196/59468977-7759dc00-8e01-11e9-963e-477c053a837c.png)

Reply:
Yeah, I feel like it's preferable.